Sprint-2/extend/format-time.js

```diff
@@ -21,4 +28,4 @@ const targetOutput2 = "11:00 pm";
 console.assert(
   currentOutput2 === targetOutput2,
   `current output: ${currentOutput2}, target output: ${targetOutput2}`
-);
+);
```

Comment:
1. When you executed this script, does you notice any error message (caused by a bug in your program)?
2. You can add additional tests to check if your function can return the expected value when times are "00:12", "12:59".

Reply:
I Missed between hours and minutes, so I have updated it.
All these passed.
  { input: "01:00", expected: "1:00 am" },  
    { input: "12:00", expected: "12:00 pm" }, 
    { input: "23:59", expected: "11:59 pm" }, 
   { input: "15:45", expected: "3:45 pm" },  
    { input: "08:30", expected: "8:30 am" }, 
   { input: "00:00", expected: "12:00 am" },
    { input: "20:15", expected: "8:15 pm" },